packages/runtime/container-runtime/src/summary/summaryFormat.ts

```diff
@@ -99,7 +113,7 @@ export interface IContainerRuntimeMetadata extends ICreateContainerMetadata, IGC
 	/** GUID to identify a document in telemetry */
 	readonly telemetryDocumentId?: string;
 	/** True if the runtime IdCompressor is enabled */
-	readonly idCompressorEnabled?: boolean;
+	readonly idCompressorEnabled?: IdCompressorMode;
```

Comment:
As per one of my comments on containerRuntime.ts, I don't think you can change the type here because on reading the metadata, the type can be boolean from old docs.

Reply:
It was always experimental and never enabled in production, so I think I can change it.
The alternative is not great - I'd need too variables in metadata